sourcegraph/handbook
# Updates to CPT documentation

This updates our CPT documentation with our PR process and team changes.

### Test
- ensure grammar and sentences read well
- ensure copy renders properly
- ensure outbound and anchor links work
